consensus/ethash/consensus.go

```diff
@@ -17,28 +17,68 @@
 package ethash
 
 import (
+	"fmt"
 	"math/big"
 
 	"github.com/Taraxa-project/taraxa-evm/common"
 	"github.com/Taraxa-project/taraxa-evm/core/types"
 	"github.com/Taraxa-project/taraxa-evm/core/vm"
+	"github.com/Taraxa-project/taraxa-evm/taraxa/state/rewards_stats"
 )
 
 var (
-	FrontierBlockReward       = big.NewInt(5e+18) // Block reward in wei for successfully mining a block
-	ByzantiumBlockReward      = big.NewInt(3e+18) // Block reward in wei for successfully mining a block upward from Byzantium
-	ConstantinopleBlockReward = big.NewInt(2e+18) // Block reward in wei for successfully mining a block upward from Constantinople
+	TaraPrecision   = big.NewInt(1e+18)              // Tara precision
+	YieldPercentage = big.NewInt(20)                 // 20% yield
+	BlocksPerYear   = big.NewInt(365 * 24 * 60 * 15) // 365 days * 24 hours * 60 minutes * 15 (1 pbft block every 4 seconds -> 15 per minute)
```

Comment:
this needs to be from config as it is configurable value

Reply:
https://github.com/orgs/Taraxa-project/projects/2/views/6